x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/components/manual_configuration.tsx

```diff
@@ -0,0 +1,415 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState } from 'react';
+
+import {
+  EuiButton,
+  EuiButtonIcon,
+  EuiCode,
+  EuiCodeBlock,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiFieldText,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiFlyoutFooter,
+  EuiFormRow,
+  EuiLink,
+  EuiPanel,
+  EuiPopover,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  useGeneratedHtmlId,
+  EuiFlexGroup,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+
+import { TryInConsoleButton } from '@kbn/try-in-console';
+
+import { KibanaDeps } from '../../../../../../../common/types';
+
+import { AttachIndexBox } from '../../../connector_detail/attach_index_box';
+
+export interface ManualConfigurationProps {
+  connector: any;
+  connectorName: string;
+  isDisabled: boolean;
+  setConnectorName: Function;
+  setIsNextStepEnabled: Function;
+}
+
+export const ManualConfiguration: React.FC<ManualConfigurationProps> = ({
+  isDisabled,
+  connectorName,
+  setConnectorName,
+  connector,
+  setIsNextStepEnabled,
+}) => {
+  const { services } = useKibana<KibanaDeps>();
+  const [isPopoverOpen, setPopover] = useState(false);
+  const splitButtonPopoverId = useGeneratedHtmlId({
+    prefix: 'splitButtonPopover',
+  });
+  const onButtonClick = () => {
+    setPopover(!isPopoverOpen);
+  };
+
+  const closePopover = () => {
+    setPopover(false);
+  };
+
+  const [isFlyoutVisible, setIsFlyoutVisible] = useState(false);
+  const simpleFlyoutTitleId = useGeneratedHtmlId({
+    prefix: 'simpleFlyoutTitle',
+  });
+  const cliCode = `./bin/connectors connector create \  
```

Comment:
ideally you should find the closest constants file and put this there.

Reply:
Looking for some other examples with the nearest code I spotted this constants file, should I use this one? `kibana/x-pack/plugins/enterprise_search/common/constants.ts`
One question, if this code snippet would be dynamic, adding some custom elements in between the content, I guess it should't be managed as constant gathering the content in a common file, isn't it?

Reply:
Done consuming `kibana/x-pack/plugins/enterprise_search/common/constants.ts` and  adding new ones here:`kibana/x-pack/plugins/enterprise_search/common/constants.ts` within a new `export const CREATE_CONNECTOR_PLUGIN`